### PR TITLE
E2efirsttest

### PR DESCRIPTION
**Pull request by NepipenkoIgor**

### CODE REVIEW

File: test-e2e/app/ambassador/ambassadors.page.e2e.ts [hunk 1]
```diff
@@ -0,0 +1,94 @@
+/**
```
Comment (valorkin) [inline test-e2e/app/ambassador/ambassadors.page.e2e.ts]: remove this comment

File: test-e2e/app/ambassador/ambassadors.page.e2e.ts [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+ * Created by vs on 4/19/16.
+ */
+
+describe('Ambassador PAge', function () {
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = 100000;
+
+
+    var lastOgranization = element(by.css('.ambassadors-organization.show>img'));
```
Comment (valorkin) [inline test-e2e/app/ambassador/ambassadors.page.e2e.ts]: use const always

File: test-e2e/app/ambassador/ambassadors.page.e2e.ts [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+    var ambassadorsPeopleCountries = element.all(by.css('.country'));
+    var ambassadorsOrganizationsIMG = element.all(by.css('.ambassadors-organization.show>img'));
+    var seeMore = element.all(by.css('.custom-button'));
+
+    var EC = protractor.ExpectedConditions;
+    var TIMEOUT = 10000;
+    browser.manage().timeouts().pageLoadTimeout(75000);
+
+    beforeAll(function () {
+        browser.get('/ambassadors');
```
Comment (valorkin) [inline test-e2e/app/ambassador/ambassadors.page.e2e.ts]: @NepipenkoIgor this one runs once? or before each?
if before each -> make it running once

File: test-e2e/app/ambassador/ambassadors.page.e2e.ts [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+        browser.wait(EC.visibilityOf(lastOgranization), TIMEOUT, 'Last organization is not visible');
```
Comment (valorkin) [inline test-e2e/app/ambassador/ambassadors.page.e2e.ts]: @NepipenkoIgor can be `wait` replaced with some `pageLoaded` event?
In worst case `application` root component can expose `ngAfterContent...` something

File: test-e2e/app/main/comparison.main.e2e.ts [hunk 5]
```diff
@@ -13,45 +13,64 @@ describe('Main Page test', function() {
...
   it('CheckAndOpenFrontDoors', function () {
     browser.get('/main');
-    browser.wait(EC.visibilityOf(frontDoors), TIMEOUT, 'Element FrontDors on MainPage is not visibility');
-    browser.wait(EC.visibilityOf(homes), TIMEOUT, 'Element FrontDors on MainPage is not visibility');
-    browser.wait(EC.visibilityOf(sofas), TIMEOUT, 'Element FrontDors on MainPage is not visibility');
-    browser.wait(EC.visibilityOf(stoves), TIMEOUT, 'Element FrontDors on MainPage is not visibility');
-    browser.wait(EC.visibilityOf(toilets), TIMEOUT, 'Element FrontDors on MainPage is not visibility');
+    browser.sleep(4000);
+    browser.wait(EC.visibilityOf(frontDoors), TIMEOUT, 'Element FrontDors on MainPage is not loaded');
+    browser.wait(EC.visibilityOf(homes), TIMEOUT, 'Element FrontDors on MainPage is not loaded');
```
Comment (valorkin) [inline test-e2e/app/main/comparison.main.e2e.ts]: error message is incorrect

File: test-e2e/app/main/comparison.main.e2e.ts [hunk 6]
```diff
@@ -13,45 +13,64 @@ describe('Main Page test', function() {
...
   });
   it('CheckAndOpenToilets', function () {
     toilets.click();
-    browser.wait(EC.visibilityOf(seeAll), TIMEOUT, 'Toilets on MainPage is not visibility');
+    browser.wait(EC.visibilityOf(seeAll), TIMEOUT, 'Toilets on MainPage is not loaded');
   });
  /* afterAll(function(){
     browser.close();
   });*/
+  afterAll(function () {
```
Comment (valorkin) [inline test-e2e/app/main/comparison.main.e2e.ts]: after all? or should be just `it`?

File: test-e2e/app/map/map.page.e2e.ts [hunk 7]
```diff
@@ -0,0 +1,198 @@
+/**
+ * Created by vs on 13/4/16.
+ */
+describe('Map Page ', function() {
+  jasmine.DEFAULT_TIMEOUT_INTERVAL = 100000;
+
+  var countryLink = new Array();
```
Comment (valorkin) [inline test-e2e/app/map/map.page.e2e.ts]: you don't need this (this === `new Array()`)

File: test-e2e/app/map/map.page.e2e.ts [hunk 8]
```diff
@@ -0,0 +1,198 @@
...
+  }); //TODO This code need for starting tests on Chrome. Finish
+
+  it ('Check country Bangladesh', function () {
+    browser.sleep(2000);
+    var elem = country.first();
+    expect(elem.getText()).toEqual('Bangladesh');
+  });
+
+  it ('Check country Colombia', function () {
+    var elem = country.get(7);
```
Comment (valorkin) [inline test-e2e/app/map/map.page.e2e.ts]: this kind of `get` will be dead as soon as content changes
but it should NOT
use smart selection (by contry id or what ever, something more persistent)

**Comment by valorkin** [comment]
1. apply comments (to all code if comment is generic, like using `const` )
2. do not ignore `eslint`
3. squash commits
   git rebase -i origin/development

@NepipenkoIgor give me a note when done

File: test-e2e/app/ambassador/ambassadors.page.e2e.ts (hunk 3 above)
Comment (ludmilanesvitiy) [inline test-e2e/app/ambassador/ambassadors.page.e2e.ts]: The beforeAll function is called only once before all the specs in describe are run, and the afterAll function is called after all specs finish.

http://jasmine.github.io/2.1/introduction.html

File: test-e2e/app/ambassador/ambassadors.page.e2e.ts (hunk 1 above)
Comment (ludmilanesvitiy) [inline test-e2e/app/ambassador/ambassadors.page.e2e.ts]: ok

File: test-e2e/app/ambassador/ambassadors.page.e2e.ts (hunk 2 above)
Comment (ludmilanesvitiy) [inline test-e2e/app/ambassador/ambassadors.page.e2e.ts]: ok

File: test-e2e/app/main/comparison.main.e2e.ts (hunk 6 above)
Comment (ludmilanesvitiy) [inline test-e2e/app/main/comparison.main.e2e.ts]: should be just it. Will change.

File: test-e2e/app/map/map.page.e2e.ts (hunk 8 above)
Comment (ludmilanesvitiy) [inline test-e2e/app/map/map.page.e2e.ts]: yes, understand. will change.

File: test-e2e/app/ambassador/ambassadors.page.e2e.ts (hunk 4 above)
Comment (NepipenkoIgor) [inline test-e2e/app/ambassador/ambassadors.page.e2e.ts]: @valorkin: I need some time for research. And then i will be able to answer for this question

**Pull request merged**